codegen_java/project/Publishing.scala

```diff
@@ -1,37 +1,24 @@
 import sbt.Keys._
 import sbt._
-import Artifactory._
```

Comment:
does this allow deleting [Artifactory.scala](https://github.com/broadinstitute/sam/blob/develop/codegen_java/project/Artifactory.scala)?

Reply:
Oh I think I meant to do that and forgot, thank you!